benchmark/url/url-searchparams-sort.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const common = require('../common.js');
+const URLSearchParams = require('url').URLSearchParams;
+
+const inputs = {
+  empty: '',
+  sorted: 'a&b&c&d&e&f&g&h&i&j&k&l&m&n&o&p&q&r&s&t&u&v&w&x&y&z',
+  almostsorted: 'a&b&c&d&e&f&g&i&h&j&k&l&m&n&o&p&q&r&s&t&u&w&v&x&y&z',
+  reversed: 'z&y&x&w&v&u&t&s&r&q&p&o&n&m&l&k&j&i&h&g&f&e&d&c&b&a',
+  random: 'm&t&d&c&z&v&a&n&p&y&u&o&h&l&f&j&e&q&b&i&s&x&k&w&r&g',
+  // 8 elements
+  short: 'm&t&d&c&z&v&a&n',
+  // 88 elements
+  long: 'g&r&t&h&s&r&d&w&b&n&h&k&x&m&k&h&o&e&x&c&c&g&e&b&p&p&s&n&j&b&y&z&' +
```

Comment:
This won't be 88 parameters because only unique keys are kept by `querystring.parse()`. One suggestion is to use unique keys first:

```js
long: 'g&r&t&h&s&rr&d&w&b&n&hh&k&x&m&kk&hhh&o&e&xx&c&cc&gg&ee&bb&' +
      'p&pp&ss&nn&j&bbb&y&z&u&l&oo&rrr&ww&a&uu&ll&mm&f&jj&q&ppp&ff&' +
      'eee&yy&eeee&nnn&eeeee&lll&mmm&www&uuu&wwww&tt&nnnn&ttt&qq&v&' +
      'yyy&ccc&ooo&kkk&fff&jjj&i&llll&mmmm&ggg&jjjj&dd&ii&zz&qqq&pppp&' +
      'xxx&qqqq&qqqqq&ddd&nnnnn&yyyy&wwwww&gggg&iii&vv&rrrr'
```

and then add before the timed loop:

```js
if (conf.type === 'long') {
  // Make `array` contain duplicate keys, useful for benchmarking stable-ness
  // of sorting
  for (i = 0; i < array.length; i += 2) {
    if (array[i].length > 1)
      array[i] = array[i][0];
  }
}
```

This should trigger the current alternate code path that uses merge sort.